sonar-java-plugin/src/main/java/org/sonar/plugins/java/CheckList.java

```diff
@@ -24,6 +24,7 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+import org.sonar.java.checks.spring.BeanMethodOfNonProxiedSingletonInvocationCheck;
 
```

Comment:
Bad sorting of `import` statements: should be moved to the block with the other spring checks imports.